lib/SILOptimizer/IPO/CrossModuleOptimization.cpp

```diff
@@ -434,13 +457,18 @@ bool CrossModuleOptimization::canSerializeFunction(
     return false;
 
   // Check if any instruction prevents serializing the function.
+  InstructionVisitor visitor(*function, *this, true);
```

Comment:
Please add an "argument label" here: `InstructionVisitor visitor(*function, *this, /*onlyCheckIfSerializable=*/ true)`